recipes/sqlitecpp/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "sqlitecpp" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/SRombauts/SQLiteCpp/archive/{{ version }}.tar.gz
+  sha256: 152673de2d0deb918974be503dc624145c2c383e02eef9356fa67c08eb4c20f4
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x' ) }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make  # [unix]
```

Comment:
doesn't this need ninja on windows?

Reply:
nmake should be fine. However, windows seems to be still broken on staged-recipes at least.